ebuild-maintenance/git/text.xml

```diff
@@ -90,7 +90,7 @@ graft it into the repository:
 </p>
 
 <pre>
-git remote add history https://anongit.gentoo.org/git/repo/gentoo/historical.git/
+git remote add history https://anongit.gentoo.org/git/archive/repo/gentoo-2.git
 git fetch history
 git replace --graft 56bd759df1d0c750a065b8c845e93d5dfa6b549d history/master
```

Comment:
Thank you. I had updated the [wiki page](https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Grafting_Gentoo_history_onto_the_active_repository) but apparently I missed this one.

The master branch in the historical repo contains one additional testing commit which shouldn't be included. There's a tag that can be used to avoid this:
```suggestion
git replace --graft 56bd759df1d0c750a065b8c845e93d5dfa6b549d cvs-final-2015-08-08
```

Reply:
Ah, neat!